From: Nikola Kotur <nikola@platform.sh>
Date: Thu, 2 Jul 2020 16:41:58 +0200
Subject: Update Platform.sh example in test data

---
 tests/test-is-public-builtin.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test-is-public-builtin.c b/tests/test-is-public-builtin.c
index e9963b5..ea3a434 100644
--- a/tests/test-is-public-builtin.c
+++ b/tests/test-is-public-builtin.c
@@ -82,8 +82,8 @@ static void test_psl(void)
 		{ ".forgot.his.name", 1, 1 },
 		{ "whoever.his.name", 0, 0 },
 		{ "whoever.forgot.his.name", 0, 0 },
-		{ "whatever.platform.sh", 1, 1 },
-		{ ".platform.sh", 1, 1 },
+		{ "whatever.platformsh.site", 1, 1 },
+		{ ".platformsh.site", 1, 1 },
 		{ "whatever.yokohama.jp", 1, 1 },
 		{ ".yokohama.jp", 1, 1 },
 		{ ".", 1, 0 }, /* special case */
